packages/migrate/migrations/svelte-4/index.js

```diff
@@ -10,7 +10,7 @@ export async function migrate() {
 		bail('Please re-run this script in a directory with a package.json');
 	}
 
-	console.log(colors.bold().yellow('\nThis will update files in the current directory\n'));
+	console.log(colors.bold().yellow('\nThis will update files in the current src/ directory\n'));
```

Comment:
Is the path configurable? 

Outside of Kit projects I've usually seen people put svelte files in `client`, `app`, `components`...

Reply:
It isn't right now. How could we go about it instead? Scan everything except `node_modules` / `.svelte-kit` or all things inside `.gitignore`? If a jsconfig/tsconfig exists and read the paths from there?

Reply:
Probably keep it simple and add a clack prompt. The automated solutions would still miss things, whereas people can just order it to do the right thing for their codebase with an input.

Reply:
Ahh, there is already a prompt for the transitions change. Great.